Casks/g/geekbench-ai.rb

```diff
@@ -0,0 +1,25 @@
+cask "geekbench-ai" do
+  version "1.0.0,101711"
+  sha256 "122543b555ecfb4eb3b46b59e282cbb153e0f06639419f6fda3d7ce7231ff91e"
+
+  url "https://cdn.geekbench.com/GeekbenchAI-#{version.csv.first}-Mac.zip"
+  name "Geekbench AI"
+  desc "Cross-platform AI benchmark to evaluate AI workload performance"
+  homepage "https://www.geekbench.com/ai/"
+
+  livecheck do
+    url :url
+    strategy :extract_plist
+  end
```

Comment:
```suggestion
  # Upstream's appcast feed is not populated but may be able to be used in the future
  # url "https://www.primatelabs.com/appcast/geekbenchai.xml"
  # strategy :sparkle, &:short_version
  livecheck do
    url "https://www.geekbench.com/ai/download/mac/"
    regex(/href=.*?GeekbenchAI[._-]v?(\d+(?:\.\d+)+)[._-]Mac\.zip/i)
  end
```

We can extract the version string from the download URL shown on the download page to avoid having to download the artifact when using the `ExtractPlist` strategy.